conjur/list/list_data.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+"""
+ListData module
+
+This module represents the DTO that holds the params the user passes in.
+We use this DTO to build the list request
+"""
+
+# pylint: disable=too-few-public-methods
+class ListData:
+    """
+    Used for organizing the the params the user passed in to execute the list command
+    """
+    def __init__(self, **argParams):
+        self.kind = argParams['kind']
+        self.inspect = argParams['inspect']
+        self.search = argParams['search']
+        self.limit = argParams['limit']
+        self.offset = argParams['offset']
+        self.acting_as = argParams['acting_as']
+
+    def list_dictify(self):
+        """
+        Method for building a dictionary from all attributes that have values
+        """
+        list_dict={}
```

Comment:
Use Dict Comprehension

{str(attr) : value for attr,value in self.__dict__.items()}